bugRzilla_review.Rmd

```diff
@@ -74,16 +74,16 @@ After considering this open your Terminal and run the command: `mysqldump -u my_
 
 The options in use are:
 <ol>
-<li>
-The `-u` flag indicates that the MySQL `username` will follow.
-</li>
-<li>
-The `-p` flag indicates we should be prompted for the `password` associated with the above username.
-`database_name` is of course the exact name of the database to export.
-</li>
-<li>
-The `>` symbol is a Unix directive for `STDOUT`, which allows Unix commands to output the text results of the issued command to another location. In this case, that output location is a file path, specified by `output_file_path`.
-</li>
+    <li>
+        The `-u` flag indicates that the MySQL `username` will follow.
+    </li>
+    <li>
+        The `-p` flag indicates we should be prompted for the `password` associated with the above username.
+    `database_name` is of course the exact name of the database to export.
+    </li>
+    <li>
+        The `>` symbol is a Unix directive for `STDOUT`, which allows Unix commands to output the text results of the issued command to another location. In this case, that output location is a file path, specified by `output_file_path`.
```

Comment:
As you mention `>` redirects the output to somewhere else. So shouldn't we want to do the opposite? As mysql doesn't have the R-bugs.sql database or it is empty?